lib/explorer/series.ex

```diff
@@ -1819,6 +1819,40 @@ defmodule Explorer.Series do
     end
   end
 
+  @doc """
+  Returns [...] #Help!
+
+  ## Examples
+
+      iex> s1 = Explorer.Series.from_list([1, 2, 3])
+      iex> Explorer.Series.is_in(s1, [1, 2])
+      #Explorer.Series<
+        Polars[3]
+        boolean [true, true, false]
+      >
+
+      iex> s2 = Explorer.Series.from_list(["a", "b", "c"])
+      iex> Explorer.Series.is_in(s2, ["a", "b"])
+      #Explorer.Series<
+        Polars[3]
+        boolean [true, true, false]
+      >
+
+      iex> s3 = Explorer.Series.from_list([1, 2, 3])
+      iex> s4 = Explorer.Series.from_list([1, 2, 4])
+      iex> Explorer.Series.is_in(s3, s4)
+      #Explorer.Series<
+        Polars[3]
+        boolean [true, true, false]
+      >
+  """
+  @doc type: :element_wise
+  def is_in(%Series{} = left, %Series{} = right),
+    do: Shared.apply_series_impl(:is_in, [left, right])
```

Comment:
Regardless of the rename, I believe we want this:

```suggestion
  def is_in(%Series{} = left, %Series{} = right) do
    if valid_for_bool_mask_operation?(left, right) do
      Shared.apply_series_impl(:is_in, [left, right])
    else
      dtype_mismatch_error("is_in/2", left, right)
    end
    do: Shared.apply_series_impl(:is_in, [left, right])
```